test/v2/models/request/createAmendHistoricNonFhlUkPropertyAnnualSubmission/HistoricNonFhlAnnualAdjustmentsSpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.createAmendHistoricNonFhlUkPropertyAnnualSubmission
+
+import play.api.libs.json.{ JsValue, Json }
+import support.UnitSpec
+import v2.models.request.common.ukPropertyRentARoom.UkPropertyAdjustmentsRentARoom
+
+class HistoricNonFhlAnnualAdjustmentsSpec extends UnitSpec {
+
+  val requestBody: HistoricNonFhlAnnualAdjustments =
```

Comment:
may rename `requestBody` to be `annualAdjustments` as not sure request body is accurate so annualAdjustments just makes a little more sense